src/Templates/ODataT4CodeGenerator.cs

```diff
@@ -3077,6 +3090,48 @@ private IEdmStringConstantExpression GetDescriptionAnnotation(IEdmVocabularyAnno
         return model.VocabularyAnnotations(this.context.EdmModel).Where(x => x.Term.Name == "Description")
             .Select(x => x.Value).FirstOrDefault() as IEdmStringConstantExpression;
     }
+
+    /// <summary>
+    /// Searches through model's vocabulary annotations and returns dictionary of term 'Revisions' annotation kinds.
+    /// </summary>
+    private IDictionary<string,string> GetRevisionsAnnotation(IEdmVocabularyAnnotatable schemaElement)
+    {
+        IEnumerable<IEdmExpression> collection = schemaElement.VocabularyAnnotations(this.context.EdmModel).Where(x => x.Term.FullName() == "Org.OData.Core.V1.Revisions")
+            .Select(x => x.Value);
+
+        IDictionary<string, string> revisionsAnnotation = new Dictionary<string, string>();
+
+        if (collection?.Any() == true)
+        {
+            IEdmCollectionExpression semanticElement = collection.FirstOrDefault() as IEdmCollectionExpression;
```

Comment:
```suggestion
        IEdmCollectionExpression semanticElement = collection?.FirstOrDefault() as IEdmCollectionExpression;
```
Since `.Any()` is always executed whenever the collection is not null, I think you can forgo that and just call `.FirstOrDefault()`

Reply:
#Resolved